website/docs/cloud-docs/workspaces/health.mdx

```diff
@@ -11,7 +11,7 @@ HCP Terraform can perform automatic health assessments in a workspace to assess
 - [Drift detection](#drift-detection) determines whether your real-world infrastructure matches your Terraform configuration.
 - [Continuous validation](#continuous-validation) determines whether custom conditions in the workspace’s configuration continue to pass after Terraform provisions the infrastructure.
 
-When enabled, HCP Terraform automatically runs a health assessment for your workspace about every 4 hours. Refer to [Health Assessment Scheduling](#health-assessment-scheduling) for details.
+When enabled, HCP Terraform periodically runs an automated health assessment for your workspace. Refer to [Health Assessment Scheduling](#health-assessment-scheduling) for details.
```

Comment:
```suggestion
When you enable health assessments, HCP Terraform periodically runs health assessments for your workspace. Refer to [Health Assessment Scheduling](#health-assessment-scheduling) for details.
```

Remove passive voice, mention the feature by name. Although now it says "health assessment" twice, I think it scans ok?